lamindb/curators/__init__.py

```diff
@@ -1712,6 +1715,80 @@ def from_tiledbsoma(
             exclude=exclude,
         )
 
+    @classmethod
```

Comment:
I'd not add the duplicated interface.

We can discuss this but I think the new way of calling curators should be
```python
ln.curators.SpatialDataCurator(sdata, ...)
```
instead of 
```python
ln.Curator.from_spatialdata()
```

Because we only want one way of doing things it should only be the `ln.curators.SpatialDataCurator(sdata, ...)` way I'd say. 🤔

Reply:
This will happen but not with this PR: https://github.com/laminlabs/lamindb/issues/2303